src/util.cc

```diff
@@ -423,21 +427,79 @@ const char* SpellcheckString(const char* text, ...) {
 }
 
 #ifdef _WIN32
+wstring Utf8ToWide(const string& u8_string) {
+  if (u8_string.empty())
+    return wstring();
+
+  size_t size_needed = MultiByteToWideChar(CP_UTF8, 0, u8_string.data(),
+                                           (int)u8_string.size(), NULL, 0);
+  if (size_needed == 0)
+    Fatal("Failed conversion to wide string");
+
+  vector<wchar_t> w_buffer(size_needed);
+  int w_size_needed =
+      MultiByteToWideChar(CP_UTF8, 0, u8_string.data(), (int)u8_string.size(),
+                          &w_buffer[0], w_buffer.size());
+  if (w_size_needed == 0)
+    Fatal("Failed conversion to wide string");
+
+  return std::wstring(&w_buffer[0], w_size_needed);
```

Comment:
This copies the string value, and incurs an additional allocation. I recommend @MrTrillian 's suggestion.